lib/hanami/cli/generators/gem/app/gemfile.erb

```diff
@@ -2,14 +2,15 @@
 
 source "https://rubygems.org"
 
-gem "rake"
-
+gem "hanami", "<%= hanami_version %>"
 gem "hanami-router", "<%= hanami_version %>"
 gem "hanami-controller", "<%= hanami_version %>"
-gem "hanami", "<%= hanami_version %>"
 
+gem "dry-types"
 gem "puma"
+gem "rake"
 
 group :cli, :development, :test do
+  gem "dotenv"
```

Comment:
@timriley `:cli` group is only for gems that enhance `hanami-cli`.

This should be:

```ruby
group :development, :test do
  gem "dotenv"
end
```

Reply:
Oh thanks, I've addressed this now! (Via a force push)